pkgs/development/python-modules/flufl/lock.nix

```diff
@@ -14,6 +14,11 @@ buildPythonPackage rec {
   propagatedBuildInputs = [ atpublic psutil ];
   checkInputs = [ pytestCheckHook pytestcov sybil ];
 
+  # disable code coverage checks for all OS. Upstream does not enforce these
+  # checks on Darwin, and code coverage cannot be imporved downstream nor is it
```

Comment:
nit: typo: "imp**or**ved"

Reply:
We don't need the comment at all. Disabling cov is a now common thing to do for python packages.